src/main/java/com/epam/grid/engine/service/JobOperationProviderService.java

```diff
@@ -89,6 +95,15 @@ public DeletedJobInfo deleteJob(final DeleteJobFilter deleteJobFilter) {
      * @return Running job.
      */
     public Job runJob(final JobOptions options) {
+        Optional.ofNullable(options.getWorkingDir()).ifPresent(workingDir -> {
+            final String absolutePath = DirectoryPathUtils
```

Comment:
it is better to name this variable like `workingDirAbsolutePath`
You see, such method like `resolvePathToAbsolute` should be as abstract as possible (`resolvePathToAbsolute` itself is a good example, it can perform action on **any** directory and provide you with its absolute path), but usage of this method here has a meaning context, that is why it is better to name it like this - to give a context on what is going on here (why abstract method is called, what it is doing) by proper naming variable

Reply:
got it. renamed